uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -793,4 +803,24 @@ private function _uninstallActivityTypes($component, array $types) {
       civicrm_api3('OptionValue', 'delete', array('id' => $id));
     }
   }
+
+  /**
+   * Renames the label for a case status
+   *
+   * @param string $fromLabel
+   * @param string $toLabel
+   */
+  private function _renameCaseStatus($fromLabel, $toLabel) {
```

Comment:
I wonder would this be more accurate if it was called `_relabelCaseStatus`? Since these are option values they also have a `name` field, which 'rename` might sound like it is changing.